packages/ckeditor5-ui/src/colorgrid/utils.js

```diff
@@ -64,13 +64,16 @@ export function normalizeColorOptions( options ) {
 }
 
 // Creates a normalized color definition from the user-defined configuration.
+// The "normalization" means it will create full
+// {@link module:ui/colorgrid/colorgrid~ColorDefinition `ColorDefinition-like`}
+// object for string values, and add a `view` property, for each definition.
 //
 // @param {String|module:ui/colorgrid/colorgrid~ColorDefinition}
 // @returns {module:ui/colorgrid/colorgrid~ColorDefinition}
 export function normalizeSingleColorDefinition( color ) {
```

Comment:
No tests for this change. (looks like should land in `normalizeColorOptions()` tests.

Sorry :)

Reply:
Do you mean, there is no test for what I added to the description? Or the fact it does not replace spaces any longer?

I think https://github.com/ckeditor/ckeditor5/pull/6801/files/4ea684d2c61271b4d8a8f6f85239a7b6d390bb1c#diff-8075f08ce8d2dc6e3b560e6c0adce83fR307 tests it

Reply:
The change is `ckeditor5-ui` and you test it in `ckeditor5-table` . The base test must be in `ckeditor5-ui`. Additional tests in `ckeditor5-table` are OK.

Reply:
OK, my bad again. It is OK.

Reply:
Thx, speaking of tests:

After the change 
https://github.com/ckeditor/ckeditor5/pull/6801/files/4ea684d2c61271b4d8a8f6f85239a7b6d390bb1c#diff-8075f08ce8d2dc6e3b560e6c0adce83fR307 

the test below

https://github.com/ckeditor/ckeditor5/blob/4ea684d2c61271b4d8a8f6f85239a7b6d390bb1c/packages/ckeditor5-font/tests/ui/colorui.js#L321-L327

Checks that `'rgb(255, 255, 255)'` is "transformed" to `'rgb(255, 255, 255)'` (no change).
Previously it checked that `'rgb(255, 255, 255)'` is transformed to `'rgb(255,255,255)'` (removed spaces).